apps/extension/src/application/idriss-send/components/send-widget.tsx

```diff
@@ -141,7 +141,7 @@ export const SendWidget = ({ widgetData }: Properties) => {
                   <Button
                     intent="primary"
                     size="medium"
-                    className="w-full"
+                    className="w-full uppercase"
                     type="submit"
                   >
                     {widgetOverrides?.sendButtonCopy ?? 'Send'}
```

Comment:
I think replacing 'Send' with 'SEND' instead of adding uppercase, there will be a more universal approach as the text can be passed by props.

Reply:
I agreed